bindings/node/lib/clientEncryption.js

```diff
@@ -86,6 +113,20 @@ module.exports = function(modules) {
       if (options.keyId) {
         contextOptions.keyId = options.keyId.buffer;
       }
+      if (options.keyAltName) {
+        const keyAltName = options.keyAltName;
+        if (options.keyId) {
+          throw new TypeError(`"options" cannot contain both "keyId" and "keyAltName"`);
+        }
+        const keyAltNameType = typeof keyAltName;
+        if (keyAltNameType !== 'string') {
+          throw new TypeError(
+            `"options.keyAltName" must be of type string, but was of type ${keyAltNameType}`
```

Comment:
This error seems inconsistent with the ones above where you're saying:
```
 `Option "keyAltNames" must be an array of string, but was of type ${typeof options.keyAltNames}.`
```

Reply:
I was mostly referring to calling it `options.keyAltName`, rather than just `keyAltName`. No need to hold up review for this nit though